tasks/sep/ink-004-enable-permissionless-game/ink.toml

```diff
@@ -0,0 +1,64 @@
+name = "Ink Sepolia"
```

Comment:
Ink sepolia is in the superchain-registry so we should be getting this from there: https://github.com/ethereum-optimism/superchain-registry/blob/main/superchain/configs/sepolia/ink.toml